python/pyspark/sql/dataframe.py

```diff
@@ -1749,7 +1749,11 @@ def describe(self, *cols: Union[str, List[str]]) -> "DataFrame":
         |    min|                 2|
         |    max|                 5|
         +-------+------------------+
-        >>> df.describe().show()
```

Comment:
Can we improve this example by showing other supported types only?

Reply:
I think the example above shows the only supported type column (`age`)

Do you think maybe we need to add another example with another DataFrame ?